mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -993,33 +976,24 @@ def _map_dictionary_values(
             event[mapping_dictionary][name] = value
 
 
-class WriteToKV(mlrun.feature_store.steps.MapClass):
```

Comment:
you double checked its not used anywhere else?

Reply:
Yes, the new name of this step is `UpdateEndpoint` because it can also write into a SQL table...